index.js

```diff
@@ -1220,7 +1220,7 @@ instance.prototype.action = function(action) {
 			break;
 
 		case 'speedPset':
-			cmd ='\x81\x01\x7E\x01\x0B' + String.fromCharCode(parseInt(opt.val,16) & 0xFF) + String.fromCharCode(parseInt(opt.speed,16) & 0xFF) + '\xFF';
+			cmd ='\x81\x01\x06\x01' + String.fromCharCode(parseInt(opt.val,16) & 0xFF) + String.fromCharCode(parseInt(opt.speed,16) & 0xFF) + '\xFF';
```

Comment:
What does this have to do with the rest of the changeset?

Reply:
Maybe it should have been added in a separate PR but it is the fix for bug #13 as mentioned in a comment above. It is the actual command to set the "Preset Recall Speed" in the PTZ camera, and it becomes relevant when using the preset functionality that is updated in this PR.

From the ["PTZOptics VISCA over IP Commands"](https://ptzoptics.com/wp-content/uploads/2020/08/PTZOptics-VISCA-over-IP-Rev-1_2-8-20.pdf) document:
![bild](https://user-images.githubusercontent.com/4034632/113471704-796dae80-945e-11eb-9344-b5fc03c280f4.png)